sos/report/plugins/aap_eda.py

```diff
@@ -40,14 +41,41 @@ def setup(self):
             "/etc/ansible-automation-platform/eda/server.key",
         ])
 
-        self.add_cmd_output("aap-eda-manage --version")
+        self.add_cmd_output([
+            "aap-eda-manage --version",
+            "aap-eda-manage showmigrations",
+        ])
+
         self.add_dir_listing([
             "/etc/ansible-automation-platform/",
             "/var/log/ansible-automation-platform/",
         ], recursive=True)
 
-        self.add_cmd_output("su - eda -c 'export'",
-                            suggest_filename="eda_export")
+        self.add_cmd_output("su - eda -c 'env'",
+                            suggest_filename="eda_environment")
```

Comment:
Why drop the `export` collection? Is it replaced by `env`? Was it ever correct? If it's a version dependent collection we typically carry both for some time - invalid collections due to command syntax are handled gracefully by sos.